superset-frontend/src/dashboard/reducers/types.ts

```diff
@@ -34,6 +34,16 @@ export type ChartConfiguration = {
   };
 };
 
+export type User = {
+  email: string;
+  firstName: string;
+  isActive: boolean;
+  lastName: string;
+  permissions: Record<string, any>;
+  roles: Record<string, any>;
+  userId: number;
+  username: string;
+};
```

Comment:
We already have `User` type in `src/types/bootstrapTypes.ts`. Are they the same thing?